.github/workflows/ci.yml

```diff
@@ -14,6 +14,12 @@ on:
 jobs:
   lint:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node: [16.x, 18.x]
```

Comment:
We needn't add `strategy` here, since one time is enough for linting.

Reply:
So let's set it to `18.x`?

Reply:
yeah